lcplants.py

```diff
@@ -1,8 +1,11 @@
 import csv
+import io
```

Comment:
I don't know why you weren't importing this before, but my Python 3.6.4 was complaining about the lack of it.